src/Helpers/SearchableFinder.php

```diff
@@ -56,6 +61,8 @@ public function __construct($app)
      */
     public function fromCommand(Command $command): array
     {
+        $this->command = $command;
```

Comment:
I rather not persist state on the instance between two functions. Can you pass the command on the `find` method? Also, do we have tests for this? If not, can you add?

Reply:
Cool, did that.

I initially tried adding a test, but I'm just too unfamiliar with the code base (how all the mocks work) to get that working properly.

Perhaps you can have a look? You should have access to the repo.